README.md

```diff
@@ -1,6 +1,6 @@
 # Qiskit docs
 
-The documentation content home for https://docs.quantum-computing.ibm.com. Note this repo does not contain content for https://learning.quantum-computing.ibm.com/ or https://qiskit.org. Refer to https://github.com/Qiskit/qiskit to make changes to the docs at https://qiskit.org/documentation.
```

Comment:
Bad change

Reply:
`docs.` to relative urls, otherwise leave as they are

Reply:
But, in this case it should stay as quantum-computing, since it isn't safe, and also this PR will only touch relative ones, sorry, I got a bit confused, and left some quantum.ibm.com over there